app/controllers/api/v4/rest/PrometheusController.scala

```diff
@@ -0,0 +1,38 @@
+package controllers.api.v4.rest
+
+import org.apache.pekko.stream.scaladsl.Source
+import play.api.Logging
+import play.api.http.HttpEntity.Streamed
+import play.api.mvc.{
+  AbstractController,
+  Action,
+  AnyContent,
+  ControllerComponents,
+  RangeResult,
+  ResponseHeader,
+  Result
+}
+
+import javax.inject.Inject
+
+class PrometheusController @Inject() (implicit cc: ControllerComponents)
+    extends AbstractController(cc)
+    with Logging {
+
+  val prometheusScraper = PrometheusScraper()
+
+  def metrics(): Action[AnyContent] = Action { request =>
+    logger.info("Received request to /metrics")
```

Comment:
I'd set this to `debug` log level.

```suggestion
    logger.debug("Received request to /metrics")
```

Reply:
applied